### PR TITLE
feat: 🔥 Add random code snippet

### PR DESCRIPTION
**Pull request by jasiene**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

Issue Number: #129 

## What is the new behavior?

Adds randCodeSnippet.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/falso/src/lib/code-snippet.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { codeSnippetLangToCodeSnippet } from './code-snippet-data';
+import { fake, FakeOptions } from './core/core';
+
+export type CodeSnippetLang =
```
Comment (NetanelBasal) [inline packages/falso/src/lib/code-snippet.ts:4]: Can you please leave only the most common:

  | 'bash'
  | 'c'
  | 'c#'
  | 'c++'
  | 'css'
  | 'go'
  | 'html'
  | 'java'
  | 'javascript'
  | 'php'
  | 'python'
  | 'rust'
  | 'sql'
  | 'swift';

Comment (jasiene) [reply packages/falso/src/lib/code-snippet.ts:4]: done

File: packages/falso/src/lib/code-snippet-data.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
+import type { CodeSnippetLang } from './code-snippet';
+
+type CodeSnippetLangToCodeSnippetMap = { [key in CodeSnippetLang]: string };
+
+export const codeSnippetLangToCodeSnippet: CodeSnippetLangToCodeSnippetMap = {
```
Comment (NetanelBasal) [inline packages/falso/src/lib/code-snippet-data.ts:5]: This isn't our pattern. Please use a JSON file. See [Sports](https://github.com/ngneat/falso/blob/main/packages/falso/src/lib/sports.json) for example.

File: packages/falso/src/lib/code-snippet.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * randCodeSnippet()
+ *
+ * @example
+ *
+ * randCodeSnippet({ length: 10 })
+ *
+ * @example
+ *
+ * randCodeSnippet({ lang: html }) // default: javascript
```
Comment (NetanelBasal) [inline packages/falso/src/lib/code-snippet.ts:39]:
```suggestion
 * randCodeSnippet({ lang: 'html' }) // default: javascript
```

File: packages/falso/src/lib/code-snippet-data.ts (hunk 2 above)
Comment (jasiene) [reply packages/falso/src/lib/code-snippet-data.ts:5]: The issue #129 uses template literals for the example data which aren't supported by JSON. What would you recommend instead?

Comment (NetanelBasal) [reply packages/falso/src/lib/code-snippet-data.ts:5]: use single quotes for strings or escape them.

Comment (jasiene) [reply packages/falso/src/lib/code-snippet-data.ts]: thanks, I've pushed another update

**Pull request merged**